CHANGELOG.md

```diff
@@ -13,4 +13,21 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   * Improved builds for newer compilers.
 * Maintenance
   * Adopted latest pysat development standards.
+  * Adopt pytest syntax
 
+[0.3.2] - 2022-05-13
+--------------------
+* New Features
+  * Compatible with pysat v3.0+
+* Deprecations
+* Documentation
+  * Added pull request templates and other GitHub project documentation.
+  * Switched Windows installation instructions to favor installing WSL.
+* Bug Fix
+  * Improved builds for newer compilers.
+  * Replaces uninterpretable characters with '*' so data loading may continue.
+* Maintenance
+  * Adopted latest pysat development standards.
+  * Shifted from TravisCI to GitHub Actions for online testing.
+  * Adopted setup.cfg
+  * Improved PEP8 compliance
```

Comment:
This contains a lot more changes than I see in this pull request.

Reply:
At the last release, the changelog was deleted for previous versions.  restores based on release notes.

Reply:
merged changes from main into develop-3.  This bit still shows up as a duplicate since it's a separate commit.